editor/static/css/preview.css

```diff
@@ -12,18 +12,59 @@ header {
     /*! border-bottom: 0.2rem solid hsl(0, 70%, 10%); */
     display: grid;
     color: white;
-    box-shadow: 0 0px 1px 2px hsl(0, 70%, 10%);
     z-index: 100;
-    line-height: 1.5em;
+    line-height: 0.5em;
```

Comment:
line-height shouldn't be less than 1em - it will lead to things overlapping when you don't expect them to. This happened when I made the screen very narrow. I think you really wanted to change the padding or margins around some elements.